### ISSUE TITLE
prettier and husky error

### ISSUE THREAD

**Issue by saifulkhan** [ScottishCovidResponse/rampvis-ui#126]
```sh
$ yarn lint:prettier 

Checking formatting...
.husky/post-checkout[error] No parser could be inferred for file: .husky/post-checkout
.husky/post-commit[error] No parser could be inferred for file: .husky/post-commit
.husky/post-merge[error] No parser could be inferred for file: .husky/post-merge
.husky/post-rewrite[error] No parser could be inferred for file: .husky/post-rewrite
.husky/pre-commit[error] No parser could be inferred for file: .husky/pre-commit
Dockerfile[error] No parser could be inferred for file: Dockerfile
All matched files use Prettier code style!
```

**Comment by kachkaev** [comment]
Thanks @saifulkhan! I’ll try resolving this on the weekend 🤞

**Comment by kachkaev** [comment]
Turns out that the CI was failing because of a new Prettier issue in README.md. I fixed it in c09cc2d07b4d26f69de69ba161d53d7981bbbcb6

These were actually warnings which did not affect the results of CI:
```
.husky/post-checkout[error] No parser could be inferred for file: .husky/post-checkout
.husky/post-commit[error] No parser could be inferred for file: .husky/post-commit
.husky/post-merge[error] No parser could be inferred for file: .husky/post-merge
.husky/post-rewrite[error] No parser could be inferred for file: .husky/post-rewrite
.husky/pre-commit[error] No parser could be inferred for file: .husky/pre-commit
Dockerfile[error] No parser could be inferred for file: Dockerfile
```

I got pretty confused by them, so removed the warnings via e57b0ee9c70b4c52922142553041b0631ea33376.

<img width="396" alt="Screenshot 2021-11-27 at 13 45 27" src="https://user-images.githubusercontent.com/608862/143684048-67f96b83-2481-4c3c-a9e2-40a916e55eca.png">

We also have a slightly better CI output now, thanks to 0ad3f61f8f3a0518dfa3e202ed5647a1fefc3bcb. If a check fails, a hint will suggest what to do next. Happy to improve the hints based on feedback!

**Comment by kachkaev** [comment]
While I was fiddling with linting, I realised that we fixed all typescript errors reported previously. How about making `lint:tsc` compulsory for PRs? See https://github.com/ScottishCovidResponse/rampvis-ui/pull/129
